src/app/content/components/Page/PageContent.tsx

```diff
@@ -60,6 +65,20 @@ export default styled(MainContent)`
         border-bottom: 1px solid ${style.focused};
       }
 
+      &.first.text {
+        &:before {
+          position: absolute;
+          top: 0;
+          left: 0;
+          content: "";
+          width: 0;
+          height: 0;
+          border-left: ${highlightIndicatorSize}rem solid ${style.focused};
```

Comment:
the size of the highlight indicator might be a good time to use `em`s since for instance figure caption text is smaller than regular text